audiophiler/__init__.py

```diff
@@ -63,62 +63,126 @@
 @app.route("/")
 @auth.oidc_auth('default')
 @audiophiler_auth
-def home(auth_dict=None):
+def home_def(auth_dict=None):
+    return redirect("/1")
+
+@app.route("/<int:page>")
+@auth.oidc_auth('default')
+@audiophiler_auth
+def home(page, auth_dict=None):
     # Retrieve list of files for templating
-    db_files = File.query.all()
+    query = File.query
+    rows = query.count()
+    rows = int(rows // 10 + bool(rows % 10))
```

Comment:
same comment on the naming of rows as your other pagination pr. this feels like it could be named better